include/manif/impl/se2/SE2Tangent_base.h

```diff
@@ -335,6 +335,17 @@ struct WEvaluator<SE2TangentBase<Derived>>
   }
 };
 
+template <typename Derived>
+struct RandomEvaluatorImpl<SE2TangentBase<Derived>>
+{
+  template <typename EigenDerived>
+  static void run(Eigen::MatrixBase<EigenDerived>& m)
+  {
+    // in [-1,1] /   in [-PI,PI]
+    m.setRandom().coeffRef(2) *= M_PI;
```

Comment:
Is it necessary to have this one liner? Readability-wise, I'd prefer a two-liner.

Reply:
No it isn't, I'll update it.

Reply:
Done.